src/index.js

```diff
@@ -5,9 +5,9 @@ import GoldifyApp from "./js/GoldifyApp";
 import reportWebVitals from "./js/utils/reportWebVitals";
 
 ReactDOM.render(
-  <React.StrictMode>
+  <React.Fragment>
     <GoldifyApp />
-  </React.StrictMode>,
+  </React.Fragment>,
```

Comment:
Is this required for the sortable arrays?

Reply:
I forgot to mention this. We should make an issue for this one, but there's PR waiting on the end of the people who made the new component I pulled in which would remove issues that are brought about (in console only) with StrictMode but not with Fragment.